sections/03_categorize.md

```diff
@@ -142,21 +142,34 @@ This indicates a potential strength of deep methods. It may be possible to
 repurpose features from task to task, improving overall predictions as the field
 tackles new challenges.
 
-Several authors have created reusable feature sets for medical terminologies using
-neural embeddings, as popularized by word2Vec [@tag:Word2Vec]. This approach
-was first used on free text medical notes by De Vine et al.
-[@doi:10.1145/2661829.2661974] with results at or better than traditional methods.
-Y. Choi et al.[@doi:10.1145/2567948.2577348] built embeddings of standardized
-terminologies, such as ICD and NDC, used in widely available administrative
-claims data. By learning terminologies for different entities in the same
-vector space, they can potentially find relationships between different
-domains (e.g. drugs and the diseases they treat). Medical claims data does not
-have the natural document structure of clinical notes, and this issue was
-addressed by E. Choi et al. [@doi:10.1145/2939672.2939823], who built
-embeddings using a multi-layer network architecture which mimics the structure
-of claims data. While promising, difficulties in evaluating the quality of
-these kinds of features and variations in clinical coding practices remain as
-challenges to using them in practice.
+Several authors have created reusable feature sets for medical terminologies
+using natural language processing (NLP) and neural embedding models, as
+popularized by Word2vec [@tag:Word2Vec]. This approach was first used on free
+text medical notes by De Vine et al. [@doi:10.1145/2661829.2661974]
+with results at or better than traditional methods. Y. Choi et al.
+[@doi:10.1145/2567948.2577348] built embeddings of standardized terminologies,
+such as ICD and NDC, used in widely available administrative claims data. By
+learning terminologies for different entities in the same vector space, they can
+potentially find relationships between different domains (e.g. drugs and the
+diseases they treat). Medical claims data does not have the natural document
+structure of clinical notes, and this issue was addressed by E. Choi et al.
+[@doi:10.1145/2939672.2939823], who built embeddings using a multi-layer network
+architecture which mimics the structure of claims data. While promising,
+difficulties in evaluating the quality of these kinds of features and variations
+in clinical coding practices remain as challenges to using them in practice.
+
+NLP has also been applied directly to EHRs to predict disease comorbidities and
+phenotypes as well as novel gene - disease associations. Gligorijevic et al
+[@doi:10.1038/srep32404] built neural embedding models and mined more than 35
+million records collected over a decade. The learned disease representations
+significantly ouperformed state-of-the-art methods, achieving close to 86%
```

Comment:
This is outside my area and I haven't read the Gligorijevic paper, but I am curious whether 86% accuracy should be considered good or good enough to be useful in this domain.  In some domains that might be useful.  In others it would be quite poor even if better that existing methods.